lib/index.js

```diff
@@ -1,9 +1,12 @@
+// @ts-nocheck
```

Comment:
@staltz can you help me out here? was getting really weird type errors

Reply:
re-enabled checking and "fixed" the warnings by adding half-assed types, but we'll probably want to still clean this up